cylc/flow/config.py

```diff
@@ -240,20 +238,27 @@ def __init__(
         work_dir: Optional[str] = None,
         share_dir: Optional[str] = None
     ) -> None:
+        """
+        Initialize the workflow config object.
+
+        Positional args:
+            workflow: workflow ID
+            fpath: workflow config file path
+             options: CLI options
```

Comment:
It's just "Args" or "Arguments":

```suggestion
        Args:
            workflow: workflow ID
            fpath: workflow config file path
            options: CLI options
```

Reply:
Fair enough (the intention was to indicate I had not attempted to document all the args, just the positional ones as singled out by @wxtim).